docs/pages/versions/unversioned/sdk/media-library.mdx

```diff
@@ -180,7 +182,16 @@ import * as MediaLibrary from 'expo-media-library';
 
 The following permissions are added automatically through this library's `AndroidManifest.xml`.
 
-<AndroidPermissions permissions={['READ_EXTERNAL_STORAGE', 'WRITE_EXTERNAL_STORAGE']} />
+<AndroidPermissions
+  permissions={[
```

Comment:
updated it since it didn't contain correct permissions

Reply:
do this updates only to the upcoming SDK? Asking if this should be backported to SDK 51.